duckplayer/duckplayer-impl/src/main/java/com/duckduckgo/duckplayer/impl/RealDuckPlayer.kt

```diff
@@ -184,15 +193,39 @@ class RealDuckPlayer @Inject constructor(
         pixelData: Map<String, String>,
     ) {
         if (!isFeatureEnabled) return
-        val duckPlayerPixelName = when (pixelName) {
-            "overlay" -> DUCK_PLAYER_OVERLAY_YOUTUBE_IMPRESSIONS
-            "play.use" -> DUCK_PLAYER_VIEW_FROM_YOUTUBE_MAIN_OVERLAY
-            "play.do_not_use" -> DUCK_PLAYER_OVERLAY_YOUTUBE_WATCH_HERE
-            else -> { null }
+        val duckPlayerPixelNames: List<Pair<DuckPlayerPixelName, PixelType>>? = when (pixelName) {
+            "overlay" -> {
+                listOf(
+                    DUCK_PLAYER_OVERLAY_YOUTUBE_IMPRESSIONS to Count,
+                    DUCK_PLAYER_OVERLAY_YOUTUBE_IMPRESSIONS_UNIQUE to Unique(),
+                )
+            }
+            "play.use" -> {
+                listOf(
+                    DUCK_PLAYER_VIEW_FROM_YOUTUBE_MAIN_OVERLAY to Count,
+                    DUCK_PLAYER_VIEW_FROM_YOUTUBE_MAIN_OVERLAY_UNIQUE to Unique(),
+                )
+            }
+            "play.do_not_use" -> {
+                listOf(
+                    DUCK_PLAYER_OVERLAY_YOUTUBE_WATCH_HERE to Count,
+                    DUCK_PLAYER_OVERLAY_YOUTUBE_WATCH_HERE_UNIQUE to Unique(),
+                )
+            }
+            "play.do_not_use.thumbnail" -> {
+                listOf(
+                    DUCK_PLAYER_THUMBNAIL_YOUTUBE_WATCH_HERE to Count,
+                    DUCK_PLAYER_THUMBNAIL_YOUTUBE_WATCH_HERE_UNIQUE to Unique(),
+                )
+            }
+            else -> {
+                Timber.d("Unknown duck player pixel: $pixelName, data: $pixelData")
+                null
+            }
         }
 
-        duckPlayerPixelName?.let {
-            pixel.fire(duckPlayerPixelName, parameters = pixelData)
+        duckPlayerPixelNames?.forEach { (duckPlayerPixelName, type) ->
+            pixel.fire(duckPlayerPixelName, if (type is Unique) mapOf() else pixelData, type = type)
```

Comment:
Still under discussion if unique pixels should contain params or not

Reply:
Latest update is that no params are needed